.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Extractors/DutchMergedExtractorConfiguration.cs

```diff
@@ -64,6 +64,8 @@ public class DutchMergedExtractorConfiguration : BaseDateTimeOptionsConfiguratio
             new Regex(DateTimeDefinitions.SingleAmbiguousTermsRegex, RegexFlags),
         };
 
+        public static readonly Regex[] TasksSkipMergeRegexes = System.Array.Empty<Regex>();
```

Comment:
Maybe rename to TasksModeMentionFilters?

Reply:
Renamed TasksSkipMergeRegexes to TasksModeMentionFilters everywhere in code.